api/modules/budgeting.js

```diff
@@ -65,6 +89,18 @@ const budgeting = module.exports = (() => {
                 external_uuid_type: 'STRIPE'
             }
         })
+        const client = await clientManagement.findClientWithExternalId({
+            id: stripeInvoice.customer
+        })
+
+        try {
+            await checkExistingPaymentsWithExternalId({ id: stripeInvoice.id })
+            client.currency = DEFAULT_STRIPE_CURRENCY //Stripe invoices will always have USD as currency
```

Comment:
This should be changing the client currency to match `stripeInvoice.currency` instead of `DEFAULT_STRIPE_CURRENCY`